twist_controller: 'arm_left'
tip_link: 'arm_left_7_link'
base_link: 'arm_left_base_link'
#output_file_path: "/home/fxm-cm_local/m-files/"
output_file_path: "/tmp/"


tracking_frame: "br"
reference_frame: "arm_left_base_link" 
endeffector_frame: "arm_left_7_link"

samples : 100
